fms/modules/feedforward.py

```diff
@@ -296,3 +301,976 @@ def forward(self, x):
         x_par = copy_to_tensor_model_parallel_region(x)
         out_par = GatedLinearUnit.forward(self, x_par)
         return reduce_from_tensor_model_parallel_region(out_par)
+
+
+"""Fused MoE kernel."""
+
+
+@triton.jit
```

Comment:
we should think about where to put triton kernels and if they should all go in one common directory, but there's enough here that I kinda think they should at least be in a separate file

Reply:
I agree, given their size, maybe someplace like `fms/triton/moe_kernels.py`? For these ones